Print_A_Bot/controls/views.py

```diff
@@ -31,3 +42,43 @@ def post(self, request):
 		pass
 
 
+class NewLightShowView(View):
+    def get(self, request):
+        context = {
+            'form': LightShowForm,
+        }
+        return render(request, 'create_lightshow.html', context)
+
+    def post(self, request):
+        form = LightShowForm(request.POST)
+
+        # revert back to blank form if not valid
+        if not form.is_valid():
+            context = {
+                'form': LightShowForm,
+            }
+            return render(request, 'create_lightshow.html', context)
+        form.save()
+        return redirect(reverse('home'))
+
+
+class NewLightShowStepView(View):
+	def get(self, request):
+		context = {
+			'form': LightShowStepForm
+		}
+		return render(request, 'create_lightshow_step.html', context)
+
+	def post(self, request):
+		form = LightShowStepForm(request.POST)
+		if not form.is_valid():
+			context = {
+				'form': LightShowStepForm
+			}
+			return render(request, 'create_lightshow_step.html', context)
+		form.save()
+		return redirect(reverse('home'))
```

Comment:
remember workout tracker, 2 buttons

1) add another step (return to same form to add another entry)
2) save and done (goto home)